### PR TITLE
GHC-110: Add ranges and decimals to lab tests

### PR DESCRIPTION
**Pull request by basrem**

### CODE REVIEW

File: configuration/concepts/lab_tests.csv [hunk 1]
```diff
@@ -1,110 +1,110 @@
...
+Uuid,Void/Retire,Same as mappings,Fully specified name:en,Fully specified name:es,Short name:es,Short name:en,Description:en,Data class,Data type,Complex data handler,Answers,Members,Allow decimals,Units,Normal low,Normal high,Absolute low,Absolute high
+718b4589-2a11-4355-b8dc-aa668a93e098,,,Negative,Negativo,Negativo,Negative,,Misc,N/A,,,,,,,,,
+7acfafa4-f19b-485e-97a7-c9e002dbe37a,,,Positive,Positivo,Positivo,Positive,,Misc,N/A,,,,,,,,,
+0a5d3d0a-eaad-4770-9d54-ae949a7e8a87,,,Normal,Normal,Normal,Normal,,Misc,N/A,,,,,,,,,
+790f812a-e827-4766-b6bd-ef51c534882f,,,Trace,Rastro,Rastro,Trace,,Misc,N/A,,,,,,,,,
+a6f85472-1bf5-4a05-bb38-7769ef3c6350,,,Small,Pequeña,Pequeña,Small,,Misc,N/A,,,,,,,,,
+3364e45c-e10d-4697-bb44-c26ff29a5e3a,,,Moderate,Moderada,Moderada,Moderate,,Misc,N/A,,,,,,,,,
+a0be77d9-3134-4877-b18a-f315f9c681f8,,,Large,Grande,Grande,Large,,Misc,N/A,,,,,,,,,
+97bc3471-a39c-44ce-8d01-f241e1418e41,,,Hematocrit,Hematocrito,Hematocrito,Hematocrit,,LabTest,Numeric,,,,Yes,%,39,52,,
+796a4dd1-a729-4d27-bea2-aa07496f303f,,,White Blood Cells,Células blancas de la sangre,Células blancas de la sangre,White Blood Cells,,LabTest,Numeric,,,,Yes,X10 *9L,5,10,,
```
Comment (reagan-meant) [inline configuration/concepts/lab_tests.csv]: Let's change the unit of '**X10 *9L**' to '**x10^9/L**'
Let's do this for other tests as well

File: configuration/concepts/lab_tests.csv [hunk 2]
```diff
@@ -1,110 +1,110 @@
...
+dcbe5eb7-5335-4411-ab0d-5632554cbb2c,,,Serum magnesium,Magnesio sérico,Magnesio sérico,Serum magnesium,,LabTest,Numeric,,,,,,,,,
+f6fd19ab-9dc3-44d7-abd4-249e9d74e2be,,,Uric acid,Ácido úrico,Ácido úrico,Uric acid,,LabTest,Numeric,,,,,mg/dl,2.4,7,,
+575a9acd-71ca-43c3-aa6f-4730ef9ae8a6,,,Total Protein,Proteina total,Proteina total,Total Protein,,LabTest,Numeric,,,,,,,,,
+d452a9f1-96d8-4fb4-b087-d57ec57edddd,,,Total Bilirubin,Bilirrubina total,Bilirrubina total,Total Bilirubin,,LabTest,Numeric,,,,,,,,,
+d838c2df-f8cc-4f5a-8154-a364eeb96868,,,Direct Bilirubin,Bilirrubina directa,Bilirrubina directa,Direct Bilirubin,,LabTest,Numeric,,,,,,,,,
+93a322ca-6fb9-40a7-9285-243c3b0e8330,,,Albumin,Albúmina,Albúmina,Albumin,,LabTest,Numeric,,,,,,,,,
+a3b0b440-4a15-4f7f-9ef9-d4c5ab5b5ec5,,,ALT,ALT,ALT,ALT,,LabTest,Numeric,,,,,,,,,
+8696ef92-4b42-45a1-b4a2-5e860b6bc311,,,AST,AST,AST,AST,,LabTest,Numeric,,,,,,,,,
+5ade6005-066c-434b-b4b6-639ebfe45f50,,,ALP,ALP,ALP,ALP,,LabTest,Numeric,,,,,,,,,
+d225f0c6-5215-476c-974b-c8fc58531c1d,,,mg/dl,Colesterol total,Colesterol total,Total Cholesterol,,LabTest,Numeric,,,,,mg/dl,0,190,,
```
Comment (reagan-meant) [inline configuration/concepts/lab_tests.csv]: Was this change intended?

File: configuration/concepts/lab_tests.csv [hunk 3]
```diff
@@ -1,110 +1,110 @@
...
+84a5df72-4916-4f92-9de7-836794950828,,,HDL,HDL,HDL,HDL,,LabTest,Numeric,,,,,,,,,
+14896245-7961-4f74-8119-28156b028a4e,,,LDL,LDL,LDL,LDL,,LabTest,Numeric,,,,,,,,,
+93f3addd-edcc-4b44-ba2e-c1419a721940,,,Triglycerides,Triglicéridos,Triglicéridos,Triglycerides,,LabTest,Numeric,,,,,,,,,
+fe1619a0-6171-41ce-b8ba-b0c4c47d880b,,,PT,PT,PT,PT,,LabTest,Numeric,,,,Yes,Seconds,9.5,15.8,,
+1fde01d8-7c48-407d-80ff-6413a6211b95,,,aPTT,aPTT,aPTT,aPTT,,LabTest,Numeric,,,,Yes,Seconds,22.6,37.6,,
```
Comment (reagan-meant) [inline configuration/concepts/lab_tests.csv]: Let's use 'seconds' rather than 'Seconds'

**Review by reagan-meant — CHANGES_REQUESTED** [review]

File: configuration/concepts/lab_tests.csv [hunk 4]
```diff
@@ -7,17 +7,17 @@ a6f85472-1bf5-4a05-bb38-7769ef3c6350,,,Small,Pequeña,Pequeña,Small,,Misc,N/A,,
 3364e45c-e10d-4697-bb44-c26ff29a5e3a,,,Moderate,Moderada,Moderada,Moderate,,Misc,N/A,,,,,,,,,
 a0be77d9-3134-4877-b18a-f315f9c681f8,,,Large,Grande,Grande,Large,,Misc,N/A,,,,,,,,,
 97bc3471-a39c-44ce-8d01-f241e1418e41,,,Hematocrit,Hematocrito,Hematocrito,Hematocrit,,LabTest,Numeric,,,,Yes,%,39,52,,
-796a4dd1-a729-4d27-bea2-aa07496f303f,,,White Blood Cells,Células blancas de la sangre,Células blancas de la sangre,White Blood Cells,,LabTest,Numeric,,,,Yes,X10 *9L,5,10,,
-48c0720a-4850-4f32-9348-c54a78bdceb5,,,Platelets,Plaquetas,Plaquetas,Platelets,,LabTest,Numeric,,,,,X10 * 9L,150,400,,
-54057e88-5668-494c-94a3-afd13dc745ea,,,Red Blood Cells,Las células rojas de la sangre,Las células rojas de la sangre,Red Blood Cells,,LabTest,Numeric,,,,Yes,X10 *12L,4,5,,
+796a4dd1-a729-4d27-bea2-aa07496f303f,,,White Blood Cells,Células blancas de la sangre,Células blancas de la sangre,White Blood Cells,,LabTest,Numeric,,,,Yes,X10 ^9L,5,10,,
```
Comment (reagan-meant) [inline configuration/concepts/lab_tests.csv]: Let's change the unit of 'X10 9L' to 'x10^9/L'
Let's do this for other tests as well

**Review by reagan-meant — CHANGES_REQUESTED** [review]

File: configuration/concepts/lab_tests.csv (hunk 4 above)
Comment (basrem) [reply configuration/concepts/lab_tests.csv]: This is done

File: configuration/concepts/lab_tests.csv (hunk 1 above)
Comment (basrem) [reply configuration/concepts/lab_tests.csv]: Done

**Comment by reagan-meant** [comment]
@basrem 
`Let's change the unit of 'X10 9L' to 'x10^9/L'
Let's do this for other tests as well` 
That hasn't been resolved
